internal/util/builder/ingress.go

```diff
@@ -53,3 +53,14 @@ func (b *IngressBuilder) WithNamespace(namespace string) *IngressBuilder {
 	b.ingress.ObjectMeta.Namespace = namespace
 	return b
 }
+
+func (b *IngressBuilder) WithAnnotations(annotations map[string]string) *IngressBuilder {
+	if b.ingress.Annotations == nil {
+		b.ingress.Annotations = annotations
+	} else {
+		for k, v := range annotations {
+			b.ingress.Annotations[k] = v
+		}
+	}
+	return b
```

Comment:
```suggestion
	if b.ingress.Annotations == nil {
		b.ingress.Annotations = annotations
		return b
	}
	for k, v := range annotations {
		b.ingress.Annotations[k] = v
	}
	return b
```
nit: like gophers like